wurst/_handles/DialogBox.wurst

```diff
@@ -0,0 +1,67 @@
+package DialogBox
+import HashMap
+import Dialog
+import LinkedList
+
+/**DialogBox lets you to create dialogs and associate the buttons with code.*/
+public class DialogBox
+
+    private static constant HashMap<button, trigger> buttonTriggers = new HashMap<button, trigger>
+    private static constant HashMap<dialog, DialogBox> dialogBoxes = new HashMap<dialog, DialogBox>
+    private constant LinkedList<button> buttons = new LinkedList<button>
+
+    private constant dialog dialogHandle = createDialog()
+    private constant trigger clickDialog = CreateTrigger()
+        ..registerDialogEvent(this.dialogHandle)
+        ..addAction(function removeClickedDialog)
+
+    private string title
+    
+
+    construct(string title)
+        this.title = title
+        this.dialogHandle.setMessage(title)
+        dialogBoxes.put(this.dialogHandle, this)
+
+    private static function removeClickedDialog()
+        let btn = GetClickedButton()
+        buttonTriggers.get(btn).execute()
```

Comment:
`buttonTriggers.get(btn)` could be null. Should check if the button has no attached listener and throw an error if so.

Reply:
It can be intended to create buttons without a listener. If you simply want a button to cancel the dialog, no code needs to be executed. 
The same goes for the quit button, though I am not sure it would even be used.